src/style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.box {
+  display: flex;
+  height: 300px;
+  min-width: 300px;
+  align-items: center;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+  flex-grow: 1;
+}
+
+.box1 {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.box2 {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.box3 {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.box4 {
+  background-color: rgb(40%, 0%, 0%);
+}
+
+.box5 {
+  background-color: rgb(20%, 0%, 0%);
+}
+
+.box6 {
+  background-color: black;
+}
+
+@media (min-width: 600px) {
+  .container {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
```

Comment:
let these styles be default for your container, so move them outside of this media rule